safe/gui/tools/shake_grid/test/test_shakemap_converter_dialog.py

```diff
@@ -60,13 +60,22 @@ def test_converting(self):
             '20131105060809',
             'output',
             'grid.xml')
+        place_path = standard_data_path(
+            'exposure',
+            'places_shakemap.shp'
```

Comment:
We also try to move away from shapefile and go for geojson. It's one file, easier to edit in PyCharm, and we can see them in GitHub easily: https://github.com/inasafe/inasafe/blob/develop/safe/test/data/gisv4/exposure/building-points.geojson

Reply:
OK, I will change the place shp to the geojson one